src/components/automated/Conversation.vue

```diff
@@ -55,7 +55,25 @@
                         saveToLocalStorage(this.chain, this.storageName);
                     }
                 }, 2000);
-            }
+            },
+            checkEmptyOption(data) {
+                for (let i = 0; i < data.replies.length; i++) {
+                    if (data.replies[i].text == "") {
+                    this.showit = true;
+                    setTimeout(() => {
+                        this.showit = false;
+                    }, 5000);
+                    return false;
+                    } else if (data.replies[i].next.replies.length > 0) {
+                    if (!this.checkEmptyOption(data.replies[i].next)) {
+                        
+                        return false;
+                    }
+                    }
+                }
+                this.handelSave()
+                return true;
+                }
         },
         created() {
             this.baseChain = loadFromLocalStorage(this.baseStorageName)||tree;
```

Comment:
could you please describe what you need in this function, maybe we can find another solution.
@yznetaX @beshoo

Reply:
We need to be sure that there is no empty options in the tree, and if there isvany we will not preform the save, an alert will present.

Reply:
i check if reply field empty that will return false and show "error message"
and if true you can save that will filter empty reply filed 
@mhdabdelrhman